src/infrastructure/services/nft/NftService.ts

```diff
@@ -7,7 +7,8 @@ const nftApi = api.injectEndpoints({
     getNfts: builder.mutation<NFT[], string>({
       query: issuer => `${endpoints.NFT}?issuer=${issuer}`,
       transformResponse: (data: { result: NFT[] }) => {
-        return data.result.slice(0, 25);
+        const filterArray = data.result.filter(nft => nft.offerPrice === 25.0);
+        return filterArray;
```

Comment:
WDYT about returning the value this way?

```suggestion
        return data.result.filter(nft => nft.offerPrice === 25.0);   
```

Reply:
nice, I will do that